keras/backend/torch/trainer.py

```diff
@@ -25,6 +26,20 @@ def __init__(self):
         self.test_function = None
         self.predict_function = None
 
+    def _should_torch_compile(self):
```

Comment:
I would actually moving this logic to `__init__` and have it override the `jit_compile` attribute value. Then the statements `if self.jit_compile:` can be kept (this is more future proof).

Reply:
We'd have to override the `jit_compile()`  getter if we wanted to move the version check logic to torch trainer's `__init__()` while still keeping the predicate in the `if` statement `self.jit_compile`.